docroot/themes/humsci/humsci_basic/src/scss/utilities/_toolbar.scss

```diff
@@ -3,6 +3,10 @@
   &-tab {
     a.ext {
       background-image: none;
+
+      &:hover {
+        background-image: linear-gradient(rgba(255, 255, 255, 0.125) 20%, transparent 200%); // reset the default background-image styles for the toolbar
```

Comment:
where does this style come from?

Reply:
This is the hover state of items in the toolbar. By applying `background-image: none` to the anchor link to override our `background-image` link styles, I overwrote Drupal's hover state styles that also use `background-image`. So I added them back.

Reply:
hmm can you write a comment about that and how colorful theme is what is causing the issue?

Reply:
Yep! 😄